src/core/public/saved_objects/saved_objects_client.ts

```diff
@@ -314,7 +313,10 @@ export class SavedObjectsClient {
           per_page: 'perPage',
           page: 'page',
         },
-        resp
+        {
+          ...resp,
+          saved_objects: resp.saved_objects.map((d) => this.createSavedObject(d)),
+        }
```

Comment:
The client-side version of the SO client is quite different from the server-side's. Most notable difference is that it returns the objects wrapped into a `SimpleSavedObject` class. Adding the `score` property on the client-side `find` results probably means more significant changes, as we would have to extend this `SimpleSavedObject` class to add the `score` property, which is a little more complex and subject to side effects than just extending a type as it's done on the server-side

As our current need for `score` is on the server-side anyway (at least atm), I'm just ignoring the `score` on the client-side `find`.

This decision is still open to discussion.

Reply:
Makes sense to me, hopefully we can change this client-side API sometime in the future.